src/common/services/NetworkSettingsService.ts

```diff
@@ -0,0 +1,28 @@
+import { NetworkConfigurationValues } from '../types/clusters';
+
+type PersistDhcpConfig = Pick<
+  NetworkConfigurationValues,
+  'ingressVip' | 'apiVip' | 'vipDhcpAllocation'
+>;
+
+const STORAGE_KEY = 'network-dhcp-settings';
+
+const NetworkSettingsService = {
+  getPersistedDhcpConfig(): PersistDhcpConfig | null {
+    const config = localStorage.getItem(STORAGE_KEY);
```

Comment:
I was thinking about using `sessionStorage` for this since once the users close the tab/window that leftover information is not relevant anymore (and actually may provide a false state)